doc/iso-embed-ignition.md

```diff
@@ -0,0 +1,30 @@
+# ISO-embedded Ignition configuration
+
+CoreOS ISO images are typically used for live-booting machines directly from a read-only storage device (e.g. a CDROM) on non-cloud platforms.
+
+First-boot provisioning via Ignition in such environments is difficult, as there are no reliable metadata endpoints, hypervisor back-channels, or writable disks.
```

Comment:
Maybe `...as there are no well-defined metadata endpoints and may not be any hypervisor back-channels or writable disks, and manual entry of an Ignition URL on the kernel command line is not ergonomic.`?